WalletWasabi.Gui/Shell/Commands/ToolCommands.cs

```diff
@@ -55,6 +57,11 @@ public ToolCommands(CommandIconService commandIconService, AvaloniaGlobalCompone
 				commandIconService.GetCompletionKindImage("Settings"),
 				settingsCommand);
 
+			TransactionBroadcasterCommand = new CommandDefinition(
+				"Broadcast Transaction",
+				commandIconService.GetCompletionKindImage("BroadcastTransaction"),
+				transationBroadcasterCommand);
```

Comment:
```suggestion
				transactionBroadcasterCommand);
```
This will fix the CI

Reply:
Done.